docs/architecture/microservices/docker-application-development-process/docker-app-development-workflow.md

```diff
@@ -379,12 +379,14 @@ services:
   sqldata:
     image: mcr.microsoft.com/mssql/server:latest
     environment:
-      - SA_PASSWORD=Pass@word
+      - SA_PASSWORD=[PLACEHOLDER]
       - ACCEPT_EULA=Y
     ports:
       - "5433:1433"
 ```
 
+[!INCLUDE [managed-identities](../../../framework/includes/managed-identities.md)]
```

Comment:
Does this make sense here? In a later file, you didn't add the include.

Reply:
Where exactly? I did omit it from several of the other's as they seemed to talk about local deployments, not cloud related.